src/main/java/com/meraphorce/controllers/UserController.java

```diff
@@ -16,12 +25,120 @@ public class UserController {
     private UserService userService;
 
     @PostMapping
-    public ResponseEntity<User> createUser(@RequestBody User user){
-        return ResponseEntity.ok(userService.createUser(user));
+    public ResponseEntity<ResponseDTO<?>> createUser(@RequestBody UserDTO user) {
+    	ResponseEntity<ResponseDTO<?>> response = null;
+    	ResponseDTO<?> responseDTO = null;
+    	UserDTO newUserData = null;
+    	
+    	newUserData = this.userService.createUser(user);
```

Comment:
You can simplify the initialization like this
UserDTO newUserData = this.userService.createUser(user);